config/routes.rb

```diff
@@ -2,4 +2,8 @@
 
   resources :people, only: [:index, :new, :create]
   root to: 'people#index'
+  get "people/search/email", to: "people#search_by_email"
```

Comment:
Why wouldn't we just set up a new RESTful controller for this?  It would make sense to make a search controller using the standard actions in the people namespace.